client/src/featureform/register.py

```diff
@@ -3252,7 +3254,13 @@ def __init__(
             insecure (bool): True if connecting to an insecure Featureform endpoint. False if using a self-signed or public TLS certificate
             cert_path (str): The path to a public certificate if using a self-signed certificate.
         """
-        super().__init__()
```

Comment:
Why remove the super __init__?

Reply:
Pretty sure this was leftover from when we separated `ResourceClient` from `Registrar`. As it stands now, there's no purpose in calling `super().__init__()`, though, if I'm not mistaken, it's benign to leave it. I can return it if you're concerned about it.

Reply:
I think you're right we can leave it out